test/blackbox-tests/test-cases/skip_vendored_dirs.t/run.t

```diff
@@ -0,0 +1,15 @@
+The project "root_project" vendor "a". And we are using "dirs" to skip the directory "a",
+now like a directory can have more than one status, so we choose one of them following
+this priority(example of "a"): "(dirs :standard \ a)" -> "(data_only_dirs a)" -> "(vendored_dirs a)".
+
+  $ dune build
```

Comment:
I like when the text describes what the supposed outcome is and why, so if the test yields a different result, it is easier to assess whether this is a bug or a deliberate change of behavior.

```suggestion
As in this case "a" is excluded using "dirs", we expect the build to fail, as it can't find library "a":

  $ dune build
```